mars_api/serializers.py

```diff
@@ -1,9 +1,91 @@
 from rest_framework import serializers
 
-from mars_api.models import Player
+from mars_api.models import Game, Player, PlayerScore
 
 
 class PlayerSerializer(serializers.ModelSerializer):
     class Meta:
         model = Player
         fields = "__all__"
+
+
+class GameSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Game
+        fields = "__all__"
+
+
+class PlayerSerializerForScore(serializers.ModelSerializer):
```

Comment:
Maybe call it something like `NotUniquePlayerSerializer`?

Reply:
Well, the serializer still prevents from duplicate nicknames, so not sure if `NotUniquePlayerSerializer` is the appropriate name for it.

Reply:
`NoUniqConstraintPlayerSerializer`? Something more descriptive than `ForScore` since it's not telling anything about why is it different